framework/src/modules/interoperability/mainchain/endpoint.ts

```diff
@@ -95,4 +95,11 @@ export class MainchainInteroperabilityEndpoint extends BaseInteroperabilityEndpo
 			result: !nameExists,
 		};
 	}
+
+	public getMainchainID(context: ModuleEndpointContext): { mainchainID: string } {
```

Comment:
the endpoint is added to the mainchain, but the issue request to add to the sidechain.
Also, if the chain is mainchain, mainchain id is obtainable from `system_getNodeInfo` because it's own chain.

Additionally, when we add new endpoint, please add the metadata on the corresponding module 🙏